common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         """
         self.database = MongoProxy(
             pymongo.database.Database(
-                pymongo.MongoClient(
+                pymongo.MongoReplicaSetClient(
```

Comment:
Why change to using the replica reads everywhere? Why not try it where we need it first in Split and see how it goes - then expand the usage later?